test/owncloud_add_test.cmake

```diff
@@ -21,6 +21,6 @@ macro(owncloud_add_test test_class additional_cpp)
     )
 
     add_definitions(-DOWNCLOUD_TEST)
-    add_definitions(-DOWNCLOUD_BIN_PATH=${CMAKE_BINARY_DIR}/bin)
+    add_definitions(-DOWNCLOUD_BIN_PATH="${CMAKE_BINARY_DIR}/bin")
```

Comment:
this is actually the way I use a lot while developing for KDE and it never raised a problem in Qt5 days. At Qt4 days I know we had to escape the strings.